docs/guides/nrfconnect_android_commissioning.md

```diff
@@ -0,0 +1,210 @@
+[nRF Connect Lock Example Application](../../examples/lock-app/nrfconnect/README.md) onto a CHIP-enabled Thread network. The instructions are also valid for [nRF Connect Lighting Example Application](../../examples/lighting-app/nrfconnect/README.md).
+
+Prerequisites:
+- 2x nRF52840 DK (PCA10056)
+  > :information_source: One nRF52840 DK can be replaced with another [OpenThread Radio Co-Processor](https://openthread.io/platforms/co-processor) compatible device like nRF52840 Dongle
+- 1x smartphone with Android 8+
+- 1x PC with Ubuntu 20.04 and Docker installed, with a spare Wi-Fi card.<br>
+  > :information_source: While the article uses Ubuntu 20.04 as a reference, all the presented steps should be achievable using other popular operating systems.
+
+The essence of the commissioning process is composed of the following pieces:
+1. CHIPTool discovers the accessory device over BLE.
+2. CHIPTool establishes a secure channel to the device over BLE, sends CHIP operational credentials and Thread provisioning data.
+3. The accessory device joins a Thread network.
+
+BLE is only used during the commissioning phase and afterwards IP connectivity between the smartphone and the accessory device is needed to send operational messages. Since a typical smartphone doesn't have Thread radio built-in, extra effort is needed to prepare full-fledged test environment. The article describes how to turn a PC with spare Wi-Fi card and nRF52840 DK into a Thread border router forwarding Wi-Fi packets to the Thread network the accessory device operates in. The below diagram shows connectivity between network components necessary to allow communication between devices running CHIPTool and Lock applications:
+
+```
+               +--------------------+
+               |    Smartphone      |
+     +---------|  Android CHIPTool  |---------+
+     |         +--------------------+         |
+     |                                        | Wi-Fi
+     |                                        |
+     |                                +---------------+  Ethernet  +----------+
+     |                                |      PC       |------------| Internet |
+     |                                +---------------+            +----------+
+     |                                        |
+ BLE |                                        | USB
+     |                                        |
+     |                                +---------------+
+     |                                |  nRF52840 DK  |
+     |                                | OpenThread RCP|
+     |                                +---------------+
+     |                                        |
+     |         +--------------------+         | Thread
+     |         |    nRF52840 DK     |         |
+     +---------|  Lock Application  |---------+
+               +--------------------+
+```
+## Building and flashing OpenThread RCP firmware
```

Comment:
Why do I need to do this? After reading "The article describes how to turn a PC with spare Wi-Fi card and nRF52840 DK into a Thread border router" I'm inclined to skip this section and go straight to "Configuring PC as a Thread border router".

Reply:
What if I changed: "The article describes how to turn a PC with spare Wi-Fi card and nRF52840 DK into a Thread border router" to "The article describes how to turn a PC with spare Wi-Fi card and nRF52840 DK **running OpenThread RCP firmware"** into a Thread border router"?

Reply:
Hm, isn't this the other way around? Because it's not about turning a PC and a DK into border router, but forming a border router using PC and DK, no?
"Thread Border Router in the CHIP project can be created using a PC with a spare Wi-Fi card and an nRF52840 DK that is running the OpenThread RCP firmware. This guide describes the required steps to achieve this."

Reply:
Maybe it's my lack of knowledge of english language, but what is the difference between turning a PC and a DK into a border router and forming a border router using a PC and a DK? I can use your suggestion, but I'll skip "in the CHIP project" as Thread Border Router is a broader concept.